docs/topics/feed-exports.rst

```diff
@@ -461,7 +461,7 @@ For instance, if your settings include::
 
 And your :command:`crawl` command line is::
 
-    scrapy crawl spidername -o dirname/%(batch_id)d-filename%(batch_time)s.json
+    scrapy crawl spidername -o 'dirname/%(batch_id)d-filename%(batch_time)s.json'
```

Comment:
I _think_ in Windows `cmd.exe` single quotes won’t work. See https://docs.scrapy.org/en/latest/intro/tutorial.html#extracting-data